darc/expression/expression_test.go

```diff
@@ -188,6 +188,50 @@ func TestParsing_RealIDs(t *testing.T) {
 	}
 }
 
+func TestParsing_Attr(t *testing.T) {
+	expr := []byte("attr:xyz_-:zys=123&sdy=234")
```

Comment:
Maybe add a test to ensure JSON-encoded attribute arguments are supported, as it might be a practical way to pass more complex data?